debian/rules

```diff
@@ -21,3 +21,11 @@ override_dh_auto_install:
 	mkdir -p debian/nfd/etc/avahi/services/
 	cp debian/avahi.service debian/nfd/etc/avahi/services/nfd.service
 
+override_dh_installinit:
+	dh_systemd_enable -pnfd --name=nfd nfd.service
+	dh_systemd_enable -pnfd --name=nfd-init nfd-init.service
+	dh_installinit -pnfd --no-start --noscripts
```

Comment:
Why are you adding `--noscripts`? `dh_installinit` already plays nice with `dh-systemd`. This would break the case of non-systemd you're trying to support.